src_erl/Communication_Layer/http_Nerlserver/src/nerlNetServer_app.erl

```diff
@@ -313,4 +313,41 @@ getHostName() ->
 %%
 
 stop(_State) ->
-    ok.
\ No newline at end of file
+    ok.
+
+% get this host ip
+getdeviceIP() ->
+    {ok, IFList} = inet:getifaddrs(),    % IFList format: [{IF_name, [{field, param},{},...]},...]
+    SubnetsList = getNerlSubnets(),
+    getdeviceIP(IFList, SubnetsList).
+
+getdeviceIP([], SubnetsList) ->
+    logger:error("No supported interface was found. Current supported interfaces list is: ~p",[SubnetsList]);
```

Comment:
Add to error log an explanation to edit the file: NerlNet_subnets_config

Reply:
done